Http/Statement.php

```diff
@@ -239,7 +239,12 @@ public function json($json = null)
         $this->request->headers->set('charset', 'utf-8');
         $this->request->headers->set('accept', 'application/json');
         if ($this->request->getMethod() !== "GET") {
-            $json = $json === null ? json_encode($this->request->request->all()) : (string)$json;
+            if ($json === null && !empty($this->request->request->all())) {
```

Comment:
Store $this->request->request->all() in a var

Reply:
fixed